src/signature-request/__test__/clause.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { isRight as isValid } from "fp-ts/Either";
+import { pipe } from "fp-ts/function";
+import { ClauseTitle, ClauseList } from "../clause";
+
+describe("ClauseTitle", () => {
+  it.each([
+    { title: "", expected: false },
+    { title: "Clause 1", expected: true },
+  ])("should be valid (non empty) ($#)", ({ title, expected }) => {
```

Comment:
Why not mock this data out of test? ex. in external file `src/signature-request/__mocks__/clause.ts`

Reply:
as long as the mocked data is small I prefer to keep it in the same file, to make the tests clearer

large mocks will surely be moved in a separate file